app/main.py

```diff
@@ -1,3 +1,58 @@
+from __future__ import annotations
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
```

Comment:
It's a good practice to include a docstring in the constructor to describe its purpose and parameters. This will make your code more readable and maintainable.